doc/major_minor_release_instructions.md

```diff
@@ -43,75 +43,95 @@ make regenerate-test-deploy-manifests
   copied protos:
 
     ```protobuf
-    message Op1_11 {
-      pfs_1_11.PutObjectRequest object = 2;
-      pfs_1_11.CreateObjectRequest create_object = 9;
-      pfs_1_11.TagObjectRequest tag = 3;
-      pfs_1_11.PutBlockRequest block = 10;
-      pfs_1_11.CreateRepoRequest repo = 4;
-      pfs_1_11.BuildCommitRequest commit = 5;
-      pfs_1_11.CreateBranchRequest branch = 6;
-      pps_1_11.CreatePipelineRequest pipeline = 7;
-      pps_1_11.CreateJobRequest job = 8;
+    message Op1_12 {
+      pfs_1_12.PutObjectRequest object = 2;
+      pfs_1_12.CreateObjectRequest create_object = 9;
+      pfs_1_12.TagObjectRequest tag = 3;
+      pfs_1_12.PutBlockRequest block = 10;
+      pfs_1_12.CreateRepoRequest repo = 4;
+      pfs_1_12.BuildCommitRequest commit = 5;
+      pfs_1_12.CreateBranchRequest branch = 6;
+      pps_1_12.CreatePipelineRequest pipeline = 7;
+      pps_1_12.CreateJobRequest job = 8;
+      auth_1_12.SetACLRequest set_acl = 11;
+      auth_1_12.ModifyClusterRoleBindingRequest set_cluster_role_binding = 12;
+      auth_1_12.SetConfigurationRequest set_auth_config = 13;
+      auth_1_12.ActivateRequest activate_auth = 14;
+      auth_1_12.RestoreAuthTokenRequest restore_auth_token = 15;
+      enterprise_1_12.ActivateRequest activate_enterprise = 16;
+      CheckAuthToken check_auth_token = 17;
     }
     ```
 
+* Add `Op2_0 op2_0 = 7` in `message Op { ... }`. Assign the next sequential number to op2_0.
+
 Finally, run `make proto` to rebuild the protos.
 
 #### Wire in the proto changes
 
 Copy over the converters:
 
 ```shell
-cp src/server/admin/server/convert1_10.go src/server/admin/server/convert1_11.go
+cp src/server/admin/server/convert1_11.go src/server/admin/server/convert1_12.go
 ```
 
 Then update `convert1_11.go`:
 
+* Update imports for pfs, pps, auth, enterprise to point to 1.12 version. Eg. `import client/pfs` to `import client/admin/v1_12/pfs`
+* Replace all instances of `pfs.` with `pfs1_12.`
+* Replace all instances of `pps.` with `pps1_12.`
+* Replace all instances of `auth.` with `auth1_12.`
+* Replace all instances of `enterprise.` with `enterprise1_12.`
+
+Then update `convert1_12.go`:
+
+* Replace all instances of `1_12` with `2_0`
 * Replace all instances of `1_11` with `1_12`
-* Replace all instances of `1_10` with `1_11`
 * Make any appropriate changes necessary for the conversion process (sometimes
   just the above replacements are sufficient)
 
 Update the admin client (`src/client/admin.go`) and admin server
 (`src/server/admin/server/api_server.go`). Mostly this just involves looking
-for instances of `1_10` or `1_11` in those files and updating the code.
+for instances of `1_11` or `1_12` in those files and updating the code and 
+add `applyOp2_0` function which should be a copy of `applyOp1_12`
 
-Update `convert1_7.go`, around line 309, to include `convert1_11Objects`.
+Update `convert1_7.go`, around line 309, to include `convert1_12Objects`.
```

Comment:
"...to include `convert2_0Objecs`"?

Reply:
No. This conversion is for 1_12. 2.0 conversion does not exist yet. We are getting master ready for eventually writing the 2.0 conversion when we do 2.x branch.